Source/Other/Extensions/NumberFormatterExtension.swift

```diff
@@ -9,11 +9,18 @@
 import Foundation
 
 extension NumberFormatter {
+	@objc static let decimalStyleFormatter = decimal()
 
-	@objc public static var decimalStyleFormatter: NumberFormatter = {
-			
+	@objc static func decimal() -> NumberFormatter {
```

Comment:
Why this change though? the `decimal()` is not used anywhere except above, and the new method below doesn't use it... so why not just adding the new one?

Reply:
Oh, yes you are right. I'll update it, I was testing out some stuff, reverted it but I forgot to revert this part